mesos_master/datadog_checks/mesos_master/mesos_master.py

```diff
@@ -237,6 +237,7 @@ def _check_leadership(self, url, tags=None):
 
         if state_metrics is not None:
             self.version = [int(i) for i in state_metrics['version'].split('.')]
+            self._collect_metadata(state_metrics)
```

Comment:
Since we already get the version, maybe we can do something like this so we can avoid duplicating code:

```

        if state_metrics is not None:
            version = state_metrics['version']
            self.version = [int(i) for i in version.split('.')]
            self.set_metadata('version', version)
```